File: C++/table.cpp
#include<iostream>
using namespace std;
int main()
{
	int num,counter,result;
	cout<<"Enter no :";
	cin>>num;
	for(counter=1;counter<=10;counter++)
	{
		result=num*counter;
		cout<<num<<"x"<<counter<<"="<<result<<endl;
	}
	return 0;
	
}

File: Csharp/Table.cs
using System;


namespace table
{
    class Program
    {
        static void Main()
        {
            int num, cnt, result;
            Console.WriteLine("Enter Number : ");
            num = Convert.ToInt32(Console.ReadLine());

            for(cnt=0;cnt<=10;cnt++)
            {
                result = num * cnt;
                Console.WriteLine("" + num + "*" + "" + cnt + "=" + "" + result);
            }
            Console.ReadKey();
        }
    }
}
